platform/src/main/java/org/stellar/anchor/platform/utils/ExponentialBackoffUtil.java

```diff
@@ -0,0 +1,50 @@
+package org.stellar.anchor.platform.utils;
+
+import org.stellar.anchor.util.Log;
+
+/**
+ * ExponentialBackoffUtil is used to do an exponential back-off, where the sleep time is doubled
+ * every time, until things succeed and the `resetSleepSeconds()` method is called.
+ */
+public class ExponentialBackoffUtil {
```

Comment:
💡 `ExponentialBackoffTimer`?

Reply:
Can we add unit test?

Reply:
Updated in https://github.com/stellar/java-stellar-anchor-sdk/pull/607/commits/247705aab4bb5e0d048fba7d6ae2d9c768103249.

Reply:
Tests created in https://github.com/stellar/java-stellar-anchor-sdk/pull/607/commits/689d64f1563f2cea515bb6676b15036d1face4a9.